package.json

```diff
@@ -14,7 +14,13 @@
     "test": "craco test",
     "eject": "craco eject",
     "lint": "eslint src/lib/**/*.js",
-    "format": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --write \"**/*.js\""
+    "format": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --write \"**/*.js\"",
+    "edit-linked-package": "json -I -f ./dist/package.json -e 'this.module=\"esm/index.js\", this.main=\"cjs/index.js\"' ",
```

Comment:
can we maybe have them prefixed by `_` to know they are kind of hack for now and these ones should be called normally by running `npm run ...`?

Reply:
actually I think what I did in my next PR will solve it (check craco.config.js)